src/EnergyPlus/HeatRecovery.cc

```diff
@@ -178,7 +178,7 @@ namespace HeatRecovery {
         } else {
             //   HX is placed on a BRANCH, optional arguments are not passed in from SimAirServingZones.
             //   HX will calculate its own part-load ratio if optional HXUnitEnable flag is not present
-            HXUnitOn = true;
+            HXUnitOn = (HXPartLoadRatio > 0.0);
```

Comment:
If HXUnitOn is true then the HX gets modeled even when PLR = 0, and in this case calculates the Sensible Effectiveness and reports that data. Existing example files show SensEff = 0 when the HX is off so this change retains that result.